nsot/models.py

```diff
@@ -809,9 +809,7 @@ def subnets(self, include_networks=True, include_ips=True, direct=False,
             broadcast_address__lte=self.broadcast_address
         )
 
-    # FIXME(jathan): This entire implementation needs to be revisited. It feels
-    # as if it's too complicated. It may not be, but it feels like it is.
-    def get_next_network(self, prefix_length, num=None, as_objects=True):
+    def get_next_network(self, prefix_length, num=None, strict=False, as_objects=True):
```

Comment:
One minor detail: Let's make sure that `strict` is reflected in the docstring for this method. :)